README.md

```diff
@@ -124,19 +130,28 @@ For reference, here is a exp of a step-by-step evaluation:
 6 + 4
 10
 ```
-
+// 3*2
+// 2 + 1
+// = 9
 ## Problem Seven
 
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
-
+// 4/2 
+// 8/4
+// 2+2
+// = 4
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
+// 'cater' + 'pillar'
```

Comment:
remember these build from left to right. second step would be "cater" + "pi" + "llar" and so on.